plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -60,14 +60,21 @@
         description: |
           The MAC address used to identify the device. If the MAC address is known,
           it must be provided and cannot be modified. At least one of the following parameters is required
-          to identify the specific access point: mac_address or management_ip_address.
+          to identify the specific access point: mac_address, hostname or management_ip_address.
         type: str
         required: True
       management_ip_address:
         description: |
           The Management IP Address used to identify the device. If the Management IP Address is known,
           it must be provided and cannot be modified. At least one of the following parameters is required
```

Comment:
Can we say like below for the clarity?

       To identify the specific access point, at least one of the following parameters is required: 
       - mac_address
       - hostname
       - management_ip_address

Reply:
Updated